app/email_batching.go

```diff
@@ -107,7 +107,7 @@ func (job *EmailBatchingJob) CheckPendingEmails() {
 	// without actually sending emails
 	job.checkPendingNotifications(time.Now(), job.server.sendBatchedEmailNotification)
 
-	mlog.Debug(fmt.Sprintf("Email batching job ran. %v user(s) still have notifications pending.", len(job.pendingNotifications)))
+	mlog.Debug("Email batching job ran. user(s) still have notifications pending.", mlog.Int("user(s)", len(job.pendingNotifications)))
```

Comment:
I don't think we need structured logging in this case. It would make sense to keep Sprintf here.

Reply:
done